<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SubjectsFromDatabaseDialog</class>
 <widget class="QDialog" name="SubjectsFromDatabaseDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>148</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Subjects from cubicweb database</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="serverUrlLabel">
       <property name="text">
        <string>server url:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="server_url_LineEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="rqlRequestLabel">
       <property name="text">
        <string>rql request:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="rql_request_LineEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="groupLabel">
       <property name="text">
        <string>group</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="group_LineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="load_button">
       <property name="text">
        <string>Load</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancel_button">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancel_button</sender>
   <signal>clicked()</signal>
   <receiver>SubjectsFromDatabaseDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>347</x>
     <y>124</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>73</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
